### ISSUE TITLE
The empty editor hint should be easily toggled by right clicking on the hint

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#213814]
Can this support right click to toggle the setting to hidden?

![image](https://github.com/microsoft/vscode/assets/2193314/f314f95c-92d8-431b-b6ac-32301950c278)

**Comment by jrieken** [comment]
Re-opening because I believe just right clicking to disable isn't a great nor established gesture. 

I am all in for configurability but right clicking should show a context menu with a toggle so that I know what's happening. We support a similar flow for breadcrumbs, sticky scroll, minimap etc

**Comment by jrieken** [comment]
Terminals have this

<img width="823" alt="Screenshot 2024-06-25 at 14 32 57" src="https://github.com/user-attachments/assets/d9a049ef-4f33-4f11-a492-3d53d1a4fe3b">

**Comment by Tyriar** [comment]
I should have described it better, I meant for it to be in a context menu as otherwise it's too easy to permanently dismiss accidentally

**Comment by bpasero** [comment]
I would align this with the hint we already have for empty editors:



![Image](https://github.com/user-attachments/assets/f2b78702-647e-4007-a450-925fbe1f2e2b)

**Comment by Tyriar** [comment]
I commented the other day in standup how noisy that message has become, if anything we should be reducing the size of that message. Especially if when we install Copilot it's no longer important enough to show at all. In the same vein we're trying to keep the terminal message as small as possible. @daviddossett could you weigh in here?

**Comment by jrieken** [comment]
I agree but having the full "Start typing to dismiss or don't show this again" message doesn't seem so bad. We already have the first half and with copilot installed it is much shorter than the default message
